Competition2017/src/competition/subsystems/climbing/commands/AscendClimbingCommand.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.climbing.commands;
+
+import org.apache.log4j.Logger;
+import com.google.inject.Inject;
+import competition.subsystems.climbing.ClimbingSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class AscendClimbingCommand extends BaseCommand {
```

Comment:
It seems like we could simplify this name without losing meaning; what about `AscendCommand`?